ui/src/main/webapp/tsconfig-tsModels.json

```diff
@@ -1,19 +0,0 @@
-{
-  "compilerOptions": {
-    "target": "es5",
-    "module": "system",
-    "moduleResolution": "node",
-    "sourceMap": true,
-    "emitDecoratorMetadata": true,
-    "experimentalDecorators": true,
-    "removeComments": false,
-    "noImplicitAny": false,
-    "rootDirs": [".", "../../src/main/webapp/"],
-    "outDir": "../windup-web/app/tsModels",
-    "sourceRoot": "/windup-web/"
-  },
-  // Only compile the models
-  "exclude": [
-    "../../src/main/webapp/**"
-  ]
-}
```

Comment:
:+1:  Great, I'm happy we can get rid of old obsolete stuff :) 
(If we don't really need it.)